theme.config.ts

```diff
@@ -45,3 +45,32 @@ export const networks: Record<string, Network> = {
  * Layouts in which the footer should not be visible
  */
 export const footerLayoutBlacklist = ['cover', 'intro', 'end'];
+
+export const getSectionTitleGetter = ($slidev, rawRoutes = []) => () => {
```

Comment:
I have yet to find out what the types for these params are

Reply:
Slidev doesn't have a specific type available for either, so I'll set them to `any`